### PR TITLE
Displays the node metrics sorted by metric name

### PR DESCRIPTION
**Pull request by sabieber**

### CODE REVIEW

File: src/main/java/sirius/kernel/health/metrics/Metric.java [hunk 1]
```diff
@@ -111,6 +113,9 @@ public int compareTo(Metric o) {
         if (o.state != state) {
             return o.state.ordinal() - state.ordinal();
         }
+        if (Strings.areEqual(label, o.label)) {
```
Comment (andyHa) [inline src/main/java/sirius/kernel/health/metrics/Metric.java:116]:
```suggestion
        if (!Strings.areEqual(label, o.label)) {
```

**Review by meg1502 — APPROVED** [review]

File: src/main/java/sirius/kernel/health/metrics/Metric.java (hunk 1 above)
Comment (andyHa) [reply src/main/java/sirius/kernel/health/metrics/Metric.java]: or chain Comparators...

**Review by andyHa — APPROVED** [review]

File: src/main/java/sirius/kernel/health/metrics/Metric.java [hunk 2]
```diff
@@ -47,6 +48,7 @@ public Metric(String code, String label, double value, MetricState state, String
 
     /**
      * Returns the unique code of the metric.
+     *
      * @return the ocde of this metric.
```
Comment (idlira) [inline src/main/java/sirius/kernel/health/metrics/Metric.java:52]:
```suggestion
     * @return the code of this metric.
```

File: src/main/java/sirius/kernel/health/metrics/Metric.java [hunk 3]
```diff
@@ -111,6 +113,9 @@ public int compareTo(Metric o) {
         if (o.state != state) {
             return o.state.ordinal() - state.ordinal();
         }
+        if (!Strings.areEqual(label, o.label)) {
```
Comment (idlira) [inline src/main/java/sirius/kernel/health/metrics/Metric.java:116]: rename o -> other ?

**Review by idlira — CHANGES_REQUESTED** [review]
...

**Review by mkeckmkeck — APPROVED** [review]

**Review by idlira — APPROVED** [review]

**Pull request merged**